valorant-tracker/src/pages/home/Home.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+
+function Home() {
+  return (
+    <form>
+      <label>Username
+        <input type="text" />
```

Comment:
Please include event handlers to store inputs so it can be passed to API queries